client/client-multi/client-cpp/CMakeLists.txt

```diff
@@ -256,8 +259,9 @@ message("KAA_MAX_LOG_LEVEL=${KAA_MAX_LOG_LEVEL}")
 
 if ( NOT KAA_WITHOUT_EVENTS )
```

Comment:
Prefer use consistent cmake style. Here and below different policies are used for whitepspaces